h2o-py/h2o/estimators/deeplearning.py

```diff
@@ -1003,13 +1003,14 @@ def stopping_metric(self):
         Metric to use for early stopping (AUTO: logloss for classification, deviance for regression)
 
         One of: ``"auto"``, ``"deviance"``, ``"logloss"``, ``"mse"``, ``"rmse"``, ``"mae"``, ``"rmsle"``, ``"auc"``,
-        ``"lift_top_group"``, ``"misclassification"``, ``"mean_per_class_error"``  (default: ``"auto"``).
+        ``"lift_top_group"``, ``"misclassification"``, ``"mean_per_class_error"``, ``"custom"``, ``"custom_increasing"``
```

Comment:
still don't understand why we keep those generated filed in repo, confuses me every time

Reply:
Yeah, it has pros and cons.

@seb-h2o The main reason why we do that is the ability to clearly see the effect of a change in the generator/parameter definition in the pull request. This way it is impossible to miss an accidental change, both you and the reviewer can go over it. It also makes sure the generated code doesn't change randomly (eg. unpredictable order of methods) and you can see how the file was changed in history and what changes were made.

Another benefit is that it makes the files visible to the outside world, you can search in it and you can provide links to the source code. That is very useful sometimes. You can checkout any branch and you will see the source code without running the build (you don't even need the right environment to run the build - which can change a lot over time - I am still able to look at the source code of h2o2 even though my build is failing ;)).

Reply:
ok, you convinced me: pros look useful indeed